frontend/ui/elements/reader_menu_order.lua

```diff
@@ -120,6 +120,7 @@ local order = {
         "zsync",
         "news_downloader",
         "send2ebook",
+        "moveToArchive",
```

Comment:
This pushes more plugins to the next page, not necessarily the best move.

Reply:
ok will be last here.